### ISSUE TITLE
Using the app as a QubesOS NetVM no longer has working DNS in 2021.4

### ISSUE THREAD

**Issue by djoate** [mullvad/mullvadvpn-app#2827]
## Issue report

Operating system:  QubesOS R4.0

App version:  2021.4

### Issue description

Using the Mullvad VPN Electron app as a NetVM for other VMs no longer has working DNS in 2021.4. VMs that are using the Mullvad VM as the NetVM are now unable to resolve DNS. Doing things with IP addresses still works.

This worked in previous versions:
1. Make a standalone VM based off of fedora-33 and make it provide network
3. Install a previous version (like 2021.3) of the Mullvad VPN app in the standalone VM, start it, and connect to a server
4. Start another AppVM that uses the standalone VM as the NetVM
5. Go to https://am.i.mullvad.net/connected in the AppVM

**Comment by pinkisemils** [comment]
Is the NetVM running systemd-resolved?

**Comment by djoate** [comment]
> Is the NetVM running systemd-resolved?

Yes, `sudo systemctl status systemd-resolved` in the NetVM running Mullvad app shows that it is active.

**Comment by MarkusPettersson98** [comment]
Hi @djoate, are you still using Mullvad, QubeOS, and experiencing this exact issue?
